src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -1570,8 +1605,15 @@ bool DoorLockServer::getMaxNumberOfCredentials(chip::EndpointId endpointId, Cred
         status = emberAfPluginDoorLockGetNumberOfFaceCredentialsSupported(endpointId, maxNumberOfCredentials);
         break;
     case CredentialTypeEnum::kAliroCredentialIssuerKey:
+        status = emberAfPluginDoorLockGetNumberOfAliroCredentialIssuerKeyCredentialsSupported(endpointId, maxNumberOfCredentials);
```

Comment:
I don't get it.  If we have a delegate callback for this, why do we also have this function thing?

Reply:
you are right. i assumed that this gives the number of credentails of this type that the lock has. My bad . will fix.

Reply:
done.